src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/GetAzureRmDataLakeAnalyticsCatalogItemAclEntry.cs

```diff
@@ -0,0 +1,120 @@
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = UserOwnerCatalogItemParameterSetName, HelpMessage = "The type of the catalog item(s).")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = GroupOwnerCatalogItemParameterSetName, HelpMessage = "The type of the catalog item(s).")]
+        [PSArgumentCompleter("Database")]
+        public string ItemType { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, Mandatory = true,
+            ParameterSetName = BaseCatalogItemParameterSetName,
+            HelpMessage = "The catalog item path to search within, in the format:" +
+                          "'DatabaseName.<optionalSecondPart>.<optionalThirdPart>.<optionalFourthPart>'.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, Mandatory = true,
+            ParameterSetName = UserOwnerCatalogItemParameterSetName,
+            HelpMessage = "The catalog item path to search within, in the format:" +
+                          "'DatabaseName.<optionalSecondPart>.<optionalThirdPart>.<optionalFourthPart>'.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, Mandatory = true,
+            ParameterSetName = GroupOwnerCatalogItemParameterSetName,
+            HelpMessage = "The catalog item path to search within, in the format:" +
+                          "'DatabaseName.<optionalSecondPart>.<optionalThirdPart>.<optionalFourthPart>'.")]
+        [ValidateNotNullOrEmpty]
+        public CatalogPathInstance Path { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            string[] requiredAceTypes;
+            switch (ParameterSetName)
+            {
+                case BaseCatalogParameterSetName: case BaseCatalogItemParameterSetName:
+                    requiredAceTypes = new[] {AclType.User, AclType.Group, AclType.Other};
+                    break;
+
+                case UserOwnerCatalogParameterSetName:  case UserOwnerCatalogItemParameterSetName:
+                    requiredAceTypes = new[] {AclType.UserObj};
+                    break;
+
+                case GroupOwnerCatalogParameterSetName: case GroupOwnerCatalogItemParameterSetName:
+                    requiredAceTypes = new[] {AclType.GroupObj};
+                    break;
+
+                default: throw new ArgumentException($"Invalid parameter set: {this.ParameterSetName}");
+            }
+
+            var toReturn = DataLakeAnalyticsClient.GetCatalogItemAclEntry(Account, Path, ItemType, requiredAceTypes).Select(acl => new PSDataLakeAnalyticsAcl(acl)).ToList();
+            if (toReturn.Count == 1)
+            {
+                WriteObject(toReturn[0]);
+            }
+            else
+            {
+                WriteObject(toReturn);
```

Comment:
@idear1203 since this is a list, make sure that each object in the list is individually written to the output stream by changing this to `WriteObject(toReturn, true)`